dev/reference/tools/mydumper.md

```diff
@@ -45,6 +41,40 @@ Execute the following command to back up data from TiDB. You can add command lin
 ./bin/mydumper -h 127.0.0.1 -u root -P 4000
 ```
 
+## Dump table data concurrently
+
+### Working principle
+
+Mydumper first calculates `min(_tidb_rowid)` and `max(_tidb_rowid)`, and segments the table into chunks according to the value specified by `-r`. Then, Mydumper assigns these chunks to different threads and dumps these chunks concurrently.
+
+### Parameters
+
+- `-t` or `--threads`: specifies the number of concurrent threads (`4` by default).
+- `-r` or `--rows`: specifies the maximum number of rows in a chunk. With this parameter specified, Mydumper ignores the value of `--chunk-filesize`.
+
+### Example
+
+Following is a complete Mydumper command:
+
+{{< copyable "shell-regular" >}}
+
+```
+./bin/mydumper -h 127.0.0.1 -u root -P 4000 -r 10000 -t 4
+```
+
+### TiDB version that supports the `_tidb_rowid` index
+
+Because concurrently dumping the table data uses the implicit `_tidb_rowid` row of TiDB, TiDB must support the `_tidb_rowid` index to fully take advantage of the concurrent dump.
```

Comment:
```suggestion
Because concurrent table data dump uses the implicit `_tidb_rowid` row of TiDB, TiDB versions that support the `_tidb_rowid` index can fully take advantage of the concurrent dump.
```